lib/verk/time.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Verk.Time do
```

Comment:
Also could you add `@moduledoc` to the module, `@doc`s and `@spec`s to the functions?